freqtrade/freqtradebot.py

```diff
@@ -110,6 +112,38 @@ def worker(self, old_state: State = None) -> State:
                            nb_assets=nb_assets)
         return state
 
+    def _startup_messages(self) -> None:
+        if self.config.get('dry_run', False):
+            self.rpc.send_msg({
+                'type': RPCMessageType.WARNING_NOTIFICATION,
+                'status': 'Dry run is enabled. All trades are simulated.'
+            })
+        stake_currency = self.config['stake_currency']
+        stake_amount = self.config['stake_amount']
+        minimal_roi = self.config['minimal_roi']
+        ticker_interval = self.config['ticker_interval']
+        exchange_name = self.config['exchange']['name']
+        strategy_name = self.config.get('strategy', '')
+        self.rpc.send_msg({
+            'type': RPCMessageType.CUSTOM_NOTIFICATION,
+            'status': f'*Exchange:* `{exchange_name}`\n'
+                      f'*Stake per trade:* `{stake_amount} {stake_currency}`\n'
+                      f'*Minimum ROI:* `{minimal_roi}`\n'
+                      f'*Ticker Interval:* `{ticker_interval}`\n'
+                      f'*Strategy:* `{strategy_name}`'
+        })
+        if self.config.get('dynamic_whitelist', False):
+            top_pairs = 'top ' + str(self.config.get('dynamic_whitelist', 20))
+            specific_pairs = ''
+        else:
+            top_pairs = 'whitelisted'
+            specific_pairs = '\n' + ', '.join(self.config['exchange'].get('pair_whitelist', ''))
+        self.rpc.send_msg({
+            'type': RPCMessageType.STATUS_NOTIFICATION,
+            'status': f'Searching for {top_pairs} {stake_currency} pairs to buy and sell...\
+                          {specific_pairs}'
```

Comment:
i would use the following snipped instead of splitting the line mid-string:        
```python 
    self.rpc.send_msg({
            'type': RPCMessageType.STATUS_NOTIFICATION,
            'status': f'Searching for {top_pairs} {stake_currency} pairs to buy and sell... '
                      f'{specific_pairs}'
        })
```

this will solve the "red bar" in the web-version of telegram:

![2018-08-15-062559_556x95_scrot](https://user-images.githubusercontent.com/5024695/44131558-2b462684-a054-11e8-8976-8b806411307e.png)

new section
![2018-08-15-062608_491x104_scrot](https://user-images.githubusercontent.com/5024695/44131559-2d76da0c-a054-11e8-8699-e8900ef4b84b.png)

Reply:
thanks, will update accordingly